operations/operations/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines.md

```diff
@@ -19,7 +19,8 @@ Mattermost's voice is:
 To achieve this, we keep the following principles in mind when writing content:
 
 * Write clearly and concisely - get to the point quickly without losing valuable information
-* Use simple, plain language that is easy to understand and avoid jargon
+* Use simple, plain language that is easy to understand and family-friendly
```

Comment:
Oh, I like this addition of "family-friendly"!

Reply:
I can't take credit for it, but it's a great addition. :)